app/helpers/application_helper.rb

```diff
@@ -5,4 +5,12 @@ def order_items_count
       "Your order contains #{item_count} #{'item'.pluralize(item_count)}"
     end
   end
+
+  def order_items_id
```

Comment:
What does this do for you and when do you use it?

Reply:
Avoid errors when rendering with nil items in order. Changed implementation by not rendering order unless it exists.